tests/specs/analog/uplinkCommands.spec.js

```diff
@@ -0,0 +1,110 @@
+import {test, expect} from '@playwright/test';
+import {MainPage} from '../../objects/MainPage.js';
+import fixture from '../../fixtures/main.js';
+import {uplinkComplexCommands, uplinkSimpleCommands} from '../../fixtures/analog/uplinkCommands.js';
+
+
+const validateUplinkCommand = async ( page, format, command, isComplex = false ) => {
+    if ( isComplex ) {
+        await page.getByLabel(fixture.hardwareType.label).click();
+        await page.getByText(command.hardwareType).click();
+    }
+
+    /* eslint-disable-next-line no-nested-ternary */
+    const dumpData = format === fixture.parseMessages.format.hex
+        ? isComplex ? command.hex.dump : command.hex.dump
+        : isComplex ? command.base64 : command.base64;
+
+    await page.getByLabel(fixture.parseMessages.dump.label, {exact: true}).fill(dumpData);
+    await page.getByTestId(fixture.parseMessages.parseButton).click();
+    await page.getByLabel(fixture.logs.buttons.expandLogs).click();
+    await page.waitForTimeout(2000);
+
+    if ( isComplex ) {
```

Comment:
Why there is a difference between dump with 1 vs dump with more commands - validation must be the same, why it different? 

Only case 'isComplex' must stay, it's universal for any number of commands. Also maybe add a check for a parsed commands and commands in fixtures?

And again, complex or simple commands doesn't need if all above will be applied.

Reply:
>Why there is a difference between dump with 1 vs dump with more commands - validation must be the same, why it different?
fixed

>Also maybe add a check for a parsed commands and commands in fixtures?
this check already exists